com.aobuchow.themes.spectrum/css/preference_styles.css

```diff
@@ -12,418 +12,400 @@
+    'editor.color.header_attributes=121,171,255'
+    'editor.color.header_key=141,203,226'
+    'editor.color.header_key_bold=true'
+    'editor.color.header_osgi=141,203,226'
+    'editor.color.header_value=255,198,0'
+    'editor.color.header_value_italic=true'
+    'editor.color.instr=255,255,255'
+    'editor.color.instr_bold=true'
+    'editor.color.string=255,198,0'
+    'editor.color.string_italic=true'
+    'editor.color.tag=121,171,255'
+    'editor.color.tag_bold=true'
+    'editor.color.xml_comment=199,221,12';
 }
 
-
 /* ############################## Egit ############################## */
 
 #org-eclipse-egit-ui-StagingView StyledText,
-Form Section LayoutComposite StyledText
-{
-background-color: #1c1f22;
-color: #d8dcdf;
+Form Section LayoutComposite StyledText {
+  background-color: #1c1f22;
+  color: #d8dcdf;
 }
 
-PushResultTable-SpellcheckableMessageAreaExtension StyledText {
-background-color: #2c3135;
-color: #d8dcdf;
+PushResultTable-SpellcheckableMessageAreaExtension
+  StyledText {
+  background-color: #2c3135;
+  color: #d8dcdf;
 }
 
-
 /* ############################## JDT ############################## */
 
-
 /* TODO: are these 3 necessary? */
 BulletListBlock > StyledText {
-background-color: #394146;
-color: #b3bbc1;
+  background-color: #394146;
+  color: #b3bbc1;
 }
 
 FormatterPreferenceSectionComposite {
-background-color:#49535a;
-color:#ebedef;
-swt-titlebar-color: #c7cdd1;
+  background-color: #49535a;
+  color: #ebedef;
+  swt-titlebar-color: #c7cdd1;
 }
 
 /* MarketPlace */
 DiscoveryItem StyledText {
-    background-color: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
-    color: white;
+  background-color: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
+  color: white;
 }
 
 DiscoveryItem {
-    background-color: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
-    color: white;
+  background-color: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
+  color: white;
 }
 
 /* Found in https://git.eclipse.org/c/mpc/org.eclipse.epp.mpc.git/tree/org.eclipse.epp.mpc.ui.css/css/e4-dark_mpc.css */
 .NotificationItem,
 .NotificationItem * {
-    background-color: '#org-eclipse-ui-workbench-PRIMARY_COLOR';
-    color: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
+  background-color: '#org-eclipse-ui-workbench-PRIMARY_COLOR';
+  color: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
 }
 
 #MarketplacePage CategoryItem,
 #MarketplacePage CategoryItem > GradientCanvas,
-#MarketplacePage CategoryItem > GradientCanvas > Label {
-    /* SWT-BUG workaround: a style for background is not applied on GradientCanvas (CSS engine repaint issue) */
-    background: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
-    background-image: '#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
-    background-color:'#com-aobuchow-themes-moderndark-BACKGROUND_COLOR';
-    color: '#org-eclipse-ui-workbench-LINK_COLOR';
+#MarketplacePage
+  CategoryItem
+  > GradientCanvas
```

Comment:
This... is a bit ugly. These css properties should be moved to a dedicated file anyhow. Something like `com.aobuchow.themes.spectrum/css/plugin-styling/`